src/middleware-set-current-time.js

```diff
@@ -19,17 +19,6 @@ videojs.use('*', (player) => {
       }
 
       return time;
-    },
-
-    // Sync VHS after play requests.
-    // This specifically handles replay where the order of actions is
-    // play, video element will seek to 0 (skipping the setCurrentTime middleware)
```

Comment:
We handle looping here: https://github.com/videojs/http-streaming/blob/master/src/videojs-http-streaming.js#L379

This play part of middleware is just causing issues.

Reply:
This was for properly seeking back to 0 during a replay. We'd want to make sure that that works properly.

Reply:
Looks like the relevant test is failing.